cw-dex-astroport/tests/astroport_tests.rs

```diff
@@ -214,29 +296,49 @@ mod tests {
             .unwrap();
 
         // Query LP token balance after
-        let lp_token_balance_after =
-            cw20_balance_query(&runner, lp_token_addr, contract_addr.clone()).unwrap();
+        let lp_token_balance_after = query_asset_balance(&runner, &lp_token, &contract_addr);
 
-        // Assert that LP token balance is zero after withdrawing all liquidity
-        assert_eq!(lp_token_balance_after, Uint128::zero());
+        // Assert that LP token balance is correct
+        assert_eq!(
+            lp_token_balance_after,
+            contract_lp_token_balance - withdraw_amount
+        );
 
         // Query contract asset balances, assert that all were returned
         for asset in asset_list.into_iter() {
             let asset_balance = query_asset_balance(&runner, &asset.info, &contract_addr);
             let expected_balance = expected_out.find(&asset.info).unwrap().amount;
             assert_eq!(asset_balance, expected_balance);
         }
+
+        // Withdraw liquidity with min_out equal to zero. Should succeed.
+        let withdraw_msg = ExecuteMsg::WithdrawLiquidity {
+            amount: withdraw_amount,
+            min_out: AssetList::new(),
+        };
+        runner
+            .execute_cosmos_msgs::<MsgExecuteContractResponse>(
+                &[withdraw_msg.into_cosmos_msg(contract_addr.clone(), vec![])],
+                admin,
+            )
+            .unwrap();
+
+        // Query LP token balance after
+        let lp_token_balance_after = query_asset_balance(&runner, &lp_token, &contract_addr);
+
+        // Assert that LP token balance is zero after withdrawing all liquidity
+        assert_eq!(lp_token_balance_after, Uint128::zero());
     }
 
     fn stake_all_lp_tokens<'a, R: Runner<'a>>(
         runner: &'a R,
         contract_addr: String,
-        lp_token_addr: String,
+        lp_token_denom: String,
         signer: &SigningAccount,
     ) -> ExecuteResponse<MsgExecuteContractResponse> {
         // Query LP token balance
         let lp_token_balance =
-            cw20_balance_query(runner, lp_token_addr, contract_addr.clone()).unwrap();
+            bank_balance_query(runner, signer.address().clone(), lp_token_denom.clone()).unwrap();
```

Comment:
this assumes lp token is a native denom. which is not done in the rest of the code base. if this is intentional then let's rename the function to `stake_all_native_lp_tokens`. 

but i think we should either have tests for both native and cw20 though or remove cw20 support

Reply:
just change it to query lp_token_balance there's a helper